tests/execution/test-scalars.md

```diff
@@ -46,6 +49,21 @@ type Query {
   body:
     query: '{ any1: any(value: { test: "abc" } ), any2: any(value: "string-value") }'
 
+- method: POST
+  url: http://localhost:8000/graphql
+  body:
+    query: "{ bigInt1: bigInt(value: 32147483647), bigInt2: bigInt(value: -32147483647) }"
```

Comment:
```suggestion
    query: "{ bigInt1: bigInt(value: "32147483647"), bigInt2: bigInt(value: "-32147483647") }"
```

- Scalars should always be sent as a string. 
- The test for bigint should contain a real big int.